man/systemd-debug-generator.xml

```diff
@@ -70,6 +70,42 @@
     <citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry>.</para>
   </refsect1>
 
+  <refsect1>
+    <title>System Credentials</title>
+
+    <variablelist class='system-credentials'>
+      <varlistentry>
+        <term><varname>systemd.extra-unit.*</varname></term>
+
+        <listitem><para>Credentials prefixed with <literal>systemd.extra-unit.</literal> specify additional
+        units to add to the final system. Similarly, credentials prefixed with
+        <literal>systemd.extra-unit.</literal> specify additional units to add to the initrd. Each
+        credential must be suffixed with the full unit name including the unit extension. Its contents must
+        be a valid unit file.</para>
+
+        <para>Note that these additional units are added to both the initrd and the final system.
+        <varname>ConditionPathExists=!/etc/initrd-release</varname> can be used to make sure the unit is
+        conditioned out in the initrd.</para>
+
+        <xi:include href="version-info.xml" xpointer="v256"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>systemd.unit-dropin.*</varname></term>
+        <term><varname>systemd.unit-dropin-initrd.*</varname></term>
```

Comment:
Cmdline vars that have got an initrd-specific variant usually add prefix `rd.` to the initrd name (e.g., `systemd.unit=` and `rd.systemd.unit=`). Maybe follow the same convention?